pkg/new-ui/v1beta1/authzn.go

```diff
@@ -0,0 +1,106 @@
+package v1beta1
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/kubeflow/katib/pkg/util/v1beta1/env"
+	v1 "k8s.io/api/authorization/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// ENV variables
+var (
+	USER_HEADER  = env.GetEnvOrDefault("USERID_HEADER", "kubeflow-userid")
+	USER_PREFIX  = env.GetEnvOrDefault("USERID_PREFIX", ":")
+	DISABLE_AUTH = env.GetEnvOrDefault("APP_DISABLE_AUTH", "false")
```

Comment:
Let's keep `DISABLE_AUTH` by default as `true`.
Since currently Katib doesn't support it in standalone mode, and we should update all of our [Installs](https://github.com/kubeflow/katib/tree/master/manifests/v1beta1/installs).
For [Katib with Kubeflow install](https://github.com/kubeflow/katib/tree/master/manifests/v1beta1/installs/katib-with-kubeflow) we can make appropriate patch for the Katib UI manifests.

WDYT @kimwnasptd @apo-ger ?

Reply:
Makes sense! Fixed!